README.md

```diff
@@ -21,16 +21,30 @@ This pulls submodule dependencies from github.
 Introduction
 ==
 
-### What's New in 4.3
+### What's New in 5.0
+
+**OS Version Support**
+- Android Nougat (API 25) is fully supported in Mobile SDK 5.0.
 
 **SmartStore Enhancements**
-- To enhance performance in certain edge cases, SmartStore adds an option for serializing unusually large soup elements in external storage.
+- We have added new APIs that allow hybrid developers to create their own named databases.
+
+**SmartSync Enhancements**
+- We now allow sync down/refresh of data stored in local soups by specifying the fields to sync.
+
+**Hybrid Networking Enhancements**
+- We have replaced forcetk.mobilesdk.js with force.js. Networking in hybrid apps is now handled natively. As a result, session token refresh happens automatically.
```

Comment:
Should we also indicate that smartsync.js no longer depends on jQuery but instead uses browser promises.